roles/space_server/vars/main.yml

```diff
@@ -51,6 +51,7 @@ users:
   'k2OS': true
   'semi': true
   'signout': sudo
+  'deni': sudo
```

Comment:
I don't see why you need this to play with the jumbotron. Please remove, rebase on master and bump your uid after richard's